gmcs/tdl.py

```diff
@@ -475,6 +475,30 @@ def ordered(self):
         return True
 
 
+###########################################################################
+# A TDLelem_list corresponds to a normal list (e.g. < [ PRED "_q_rel" ] >)
+
+class TDLelem_list(TDLelem_feat):
+    def __init__(self):
+        self.child = []
+
+    def write(self):
+        if debug_write:
+            TDLwrite('list\n')
+
+        TDLwrite('< ')
+        for ch in self.child[0:1]:
```

Comment:
I also thought this bit of code looked odd, but do not that `self.child[0].write()` would raise an IndexError if the `child` list is empty.

Reply:
I always forget that Python’s slices don’t raise IndexErrors and I find that odd. It seems `if self.child: self.child[0].write()` would more accurately represent the intention, but maybe that’s beyond the scope of this PR.

Reply:
I actually don't remember now why I wrote this class. I am not using it anywhere. I thought I needed it though... But I think I should just delete it for now... Or leave it commented out as we have sometimes been doing for unfinished work, so that it can remind us of what we were about to do... But it isn't reminding me of anything now, so...